src/hooks/useSortTable.hook.js

```diff
@@ -24,11 +24,13 @@ import { ReactComponent as ArrowIcon } from 'igz-controls/images/back-arrow.svg'
 
 export const useSortTable = ({ headers, content, sortConfig }) => {
   const [direction, setDirection] = useState(null)
-  const [selectedColumnName, setSelectedColumnName] = useState(null)
+  const [selectedColumnName, setSelectedColumnName] = useState('')
   const [sortedTableContent, setSortedTableContent] = useState(content)
   const [sortedTableHeaders, setSortedTableHeaders] = useState(headers)
 
   const { allowSortBy, excludeSortBy, defaultSortBy } = sortConfig
+    ? sortConfig
+    : { allowSortBy: null, excludeSortBy: null, defaultSortBy: null }
```

Comment:
you can add default values for each variable
`const { allowSortBy=null, excludeSortBy=null, defaultSortBy=null } = sortConfig`